homeassistant/components/luftdaten/config_flow.py

```diff
@@ -30,8 +39,8 @@ class LuftDatenFlowHandler(config_entries.ConfigFlow):
     def _show_form(self, errors=None):
         """Show the form to the user."""
         data_schema = OrderedDict()
-        data_schema[vol.Required(CONF_SENSOR_ID)] = str
-        data_schema[vol.Optional(CONF_SHOW_ON_MAP, default=False)] = bool
+        data_schema[vol.Required(CONF_SENSOR_ID)] = cv.positive_int
```

Comment:
I don't think we can use complex validators, just pure types. This validation will be done with `voluptuous-serialize`.

Reply:
cv.boolean does not work, cv.positive_int seems ok. Other modules also use some complex validators (openuv uses cv.latitude)

Reply:
positive int works because of it is composed of serializable parts.